ultravox/data/text_proc.py

```diff
@@ -30,6 +30,8 @@ def format_asr_text(text: str) -> str:
     """
     remaining_words = []
     for word in text.split():
+        if word in GIGASPEECH_GARBAGE_UTTERANCE_TAGS:
```

Comment:
Do we want to throw away the entire utterance here, or just the <NOISE>?

Reply:
I think for now we throw away the whole utterance, but its subject to change depending on how we want to run our experiments.